.circleci/config.yml

```diff
@@ -462,13 +465,25 @@ jobs:
           docker pull anycable/anycable-go:${CIRCLE_TAG/v/''}-mrb
           docker pull anycable/anycable-go:${CIRCLE_TAG/v/''}-alpine
     - run:
-        name: Pushs latest Docker image
+        name: Pushs latest Docker images
         command: |
             docker tag anycable/anycable-go:${CIRCLE_TAG/v/''} anycable/anycable-go:latest
+            docker tag anycable/anycable-go:${CIRCLE_TAG/v/''} anycable/anycable-go:${CIRCLE_TAG:1:1}
+            docker tag anycable/anycable-go:${CIRCLE_TAG/v/''} anycable/anycable-go:${CIRCLE_TAG:1:3}
```

Comment:
We might forget to change this rule when we will are going to release `v1.10`.
I think we should use more stable replacing rule, something like this: 
`$(echo "$CIRCLE_TAG" | sed -E 's/^v([0-9]+)\.([0-9]+)\.([0-9]+)$/{\1}/')`
`$(echo "$CIRCLE_TAG" | sed -E 's/^v([0-9]+)\.([0-9]+)\.([0-9]+)$/{\1}.{\2}/')`

Reply:
Nice, I was looking for this spell :)